PublicOnboardingDemo/clients/UserRegistration/controllers/page/Start/widgets/RegisterButton/onClick.js

```diff
@@ -0,0 +1,28 @@
+    client.getWidget("RegisterButton").isDisabled = true;
+    
+    // copy over form information to local variable
+    var userDef = {
+        email: client.data.email,
+        first_name: client.data.first_name,
+        last_name: client.data.last_name
+    };
+    
+    var http = new Http();
+    http.setVantiqUrlForSystemResource("procedures", 		client.getNamespace());
+    http.setVantiqHeaders();
+    var args = {
+        obj:userDef
+    };
+    
+    http.execute(args,"Registration.oauthUserRegistration",function(response) {
+        console.log("SUCCESS: " + JSON.stringify(response));
+        // will change pages upon a successful registration
+        client.goToPage("WaitingPage");
+    },
+    function(errors)
+    {
+        console.log(errors);
+        alert("You have entered an invalid email. Please try a different one.");
```

Comment:
Not sure this is the right error all the time. You could parse the error from the server response, and display just the message property.